mypy.ini

```diff
@@ -0,0 +1,9 @@
+# mypy.ini
+[mypy]
+check_untyped_defs = True
+disallow_untyped_defs = False
+ignore_missing_imports = True
+disable_error_code = union-attr, attr-defined, typeddict-unknown-key, no-any-return, return, assignment
+
+[mypy-tests.*]
+ignore_errors = True
```

Comment:
Instead of ignoring the tests module, I'm thinking we just update the GitHub action to just run `mypy` on the `src` directory (e.g. `mypy src`). This is mainly so that if someone did want to `mypy` on the tests, it would not automatically ignore errors.